### PR TITLE
Add inline generate command to "The REST Adapter"

### PR DESCRIPTION
**Pull request by joostdevries**
Also added a small subtitle
Refs #410

### CODE REVIEW

File: source/models/the-rest-adapter.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ associated with each model are conventional; this means that, if you
 follow the rules, you will not need to configure the adapter or write
 any code in order to get started.
 
+#### Customizing the adapter
```
Comment (michaelrkn) [inline source/models/the-rest-adapter.md]: Let's take away the heading here - it unnecessarily breaks up the page flow, and it doesn't start something that could be considered a new section.

File: source/models/the-rest-adapter.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ associated with each model are conventional; this means that, if you
...
+
+If you do need to customize the adapter for your application, you can create
+an Application Adapter which subclasses [DS.RESTAdapter](http://emberjs.com/api/data/classes/DS.RESTAdapter.html) by running:
```
Comment (locks) [inline source/models/the-rest-adapter.md]: I thought the default in 1.13 was JSONAPIAdapter?

**Comment by michaelrkn** [comment]
Hey @joostdevries - just want to ping you on these couple comments.

File: source/models/the-rest-adapter.md (hunk 2 above)
Comment (joostdevries) [inline source/models/the-rest-adapter.md]: then this entire section needs to be fixed+renamed right?

**Comment by joostdevries** [comment]
fixed the header.

**Pull request merged**